src/x509.c

```diff
@@ -12766,18 +12766,31 @@ int wolfSSL_X509_get_signature_nid(const WOLFSSL_X509 *x)
 #endif  /* OPENSSL_EXTRA */
 
 #if defined(OPENSSL_EXTRA)
-WOLFSSL_STACK* wolfSSL_sk_X509_new(void)
+WOLFSSL_STACK* wolfSSL_sk_X509_new(WOLF_SK_COMPARE_CB(WOLFSSL_X509, cb))
```

Comment:
Let's make this function just wrap wolfSSL_sk_X509_new_null. Reduces code repetition.

Reply:
it wraps wolfSSL_sk_X509_new_null now